lib/nostrum/struct/guild.ex

```diff
@@ -113,6 +115,18 @@ defmodule Nostrum.Struct.Guild do
   """
   @type system_channel_id :: Snowflake.t() | nil
 
+  @typedoc """
+  The id of the channel that is used for rules. This is only available to guilds that
+  contain ``PUBLIC`` in `t:features/0`.
+  """
+  @type rules_channel_id :: Snowflake.t() | nil
```

Comment:
While this is correct, I think it's more helpful if we link to the remote types in the actual `Nostrum.Struct.Channel` module itself. This way we signify in the type that this is an `id` coming from a channel and the user is only one click away from going to check out the type and any intricacies it may have!